### ISSUE TITLE
[FEATURE] Return message id on sendText

### ISSUE THREAD

**Issue by smashah** [smashah/sulla#166]

**Comment by joaomirandasa** [comment]
This will be really helpful!

**Comment by joaomirandasa** [comment]
One idea i have here: 
You can create mensageID before create template of message and inject this new ID into message, so when it's inside an function you will be able to return all template message (with new id) after send it.

**Comment by smashah** [comment]
I've tried a simpler way, Hopefully it works just as well.

**Comment by smashah** [comment]
Tested, working.
![image](https://user-images.githubusercontent.com/5903821/75828536-ff26b280-5da3-11ea-9232-347f5ef06938.png)

```javascript
const sentMessageId = await client.sendText(message.from, message.body);
console.log("start -> sentMessageId", sentMessageId);
```

**Comment by smashah** [comment]
@joaomirandasa took your suggestion, much more reliable. Available in v1.8.10

**Comment by joaomirandasa** [comment]
Glad to help you at this project!
